### DISCUSSION TITLE
Deno Support

### DISCUSSION THREAD

**Discussion by jlorezz** [prisma/prisma#6199] [Q&A]
Hey Guys!
How's Deno support looking right now for Prisma? I saw some other issues, and discussions on here but they are all a year or so back. What's the state of Deno support as of now? Any new news?
All the best Florian

**Comment by ruheni** [comment]
Hi @jlorezz 👋🏾

We still don't support Deno. It is still far off. Issue #2452 provides more details regarding its status.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
